packages/cli/src/platform/PlatformCommand.ts

```diff
@@ -9,14 +9,13 @@ export class PlatformCommand implements Command {
 
   public async parse(argv: string[]) {
     const isHasEarlyAccessFeatureFlag = Boolean(argv.find((_) => _.match(/early-access-feature/)))
-    // TODO: throw instead and adapt tests
-    if (!isHasEarlyAccessFeatureFlag) return new EarlyAccessFlagError() // prettier-ignore
+    if (!isHasEarlyAccessFeatureFlag) throw new EarlyAccessFlagError() // prettier-ignore
```

Comment:
Note it would be good to avoid disabling prettier completely like here for this line with "// prettier-ignore"

Reply:
I prefer to keep guards on a single line. Do you want to avoid that style?

Reply:
Using `// prettier-ignore` completely disables prettier for this line, so it's better to avoid when I think the value is low, like here.
The less `// prettier-ignore` in the codebase, the better.
There are some cases where it's actually required, these are rare.